doc/authentication-methods/anonymous.md

```diff
@@ -30,5 +30,5 @@ Specifies the SASL mechanisms supported by the `anonymous` authentication method
 
```

Comment:
In line 17 up there, could you also update the example to `protocol = ...`?

Reply:
Excellent catch, missed that one.